src/js/utils/generate_filename.js

```diff
@@ -1,6 +1,6 @@
 import semver from "semver";
 import FC from "../fc";
-import { API_VERSION_1_45 } from "../data_storage";
+import { API_VERSION_1_46 } from "../data_storage";
```

Comment:
❤️

Reply:
Usually the bump version is done by ours, but this is a draft so I think is needed to see how the code looks. When final version of Configurator is released, we will bump the version and a rebase will fix the issue.

Reply:
I just like ppl using modules 🤓

Reply:
I rolled back this change